src/Term/Fingerprint.php

```diff
@@ -278,4 +278,17 @@ public function setAliasGroups( AliasGroupList $groups ) {
 		$this->aliasGroups = $groups;
 	}
 
+	/**
+	 * @see EntityDocument::__clone
+	 *
+	 * @since 5.0
+	 */
+	public function __clone() {
+		// TermList is mutable, but Term is not. No deeper cloning necesarry.
+		$this->labels = clone $this->labels;
+		$this->descriptions = clone $this->descriptions;
+		// AliasGroupList is mutable, but AliasGroup is not. No deeper cloning necesarry.
```

Comment:
As above, you wouldn't do deeper cloning at this place but in `TermList` resp. `AliasGroupList`.